ibmdbpy/base.py

```diff
@@ -1343,8 +1338,9 @@ def append(self, idadf, df, maxnrow=None):
         """
         # SANITY CHECK : maxnrow
         if maxnrow is None:
-            # Note : 8000 is an empirical maximum number of cells
-            maxnrow = int(8000 / len(df.columns))
+            # Note : previously, int(8000 / len(df.columns)) was an empirical value for the maximum number of cells
+            # new empirical best performing value is int(100000 / len(df.columns))
+            maxnrow = int(100000 / len(df.columns))
```

Comment:
8000 also shows up in the docstring of the function.

Reply:
modified